src/extensions/default/CodeFolding/Prefs.js

```diff
@@ -8,27 +8,35 @@
 /*global define, brackets*/
 define(function (require, exports, module) {
     "use strict";
-    var PreferencesManager      = brackets.getModule("preferences/PreferencesManager"),
-        prefs                   = PreferencesManager.getExtensionPrefs("code-folding"),
-        strings                 = brackets.getModule("strings"),
-        store                   = {},
-        foldsKey                = "folds";
+    var PreferencesManager          = brackets.getModule("preferences/PreferencesManager"),
+        prefs                       = PreferencesManager.getExtensionPrefs("code-folding"),
+        foldsKey                    = "code-folding.folds",
+        // preference key strings are here for now since they are not used in any UI
+        ENABLE_CODE_FOLDING         = "Enable code folding",
+        MIN_FOLD_SIZE               = "Minimum fold size",
+        MIN_FOLD_SIZE_HELP          = "Minimum number of lines to allow in a foldable range",
+        SAVE_FOLD_STATES            = "Save fold states",
+        SAVE_FOLD_STATES_HELP       = "Save fold states to disk when editor is closed and restore the folds when reopened",
+        ALWAYS_USE_INDENT_FOLD      = "Always use indent fold",
+        ALWAYS_USE_INDENT_FOLD_HELP = "Fall back to using level of indentation as a folding guideline if no range finder is found for the current mode.",
+        FADE_FOLD_BUTTONS           = "Fade fold buttons",
+        FADE_FOLD_BUTTONS_HELP      = "Hides the fold buttons unless the mouse is over the gutter",
+        MAX_FOLD_LEVEL              = "Max fold level",
+        MAX_FOLD_LEVEL_HELP         = "Used to limit the number of nested folds to find and collapse when View -> Collapse All is called or Alt is held down when collapsing. Should improve performance for large files.";
```

Comment:
This feels a little weird -- might be better for these to just be doc comments on the `definePreference()` calls for now... but not a big deal.  I think this would be ok to merge as-is.